native_client/BUILD

```diff
@@ -1,30 +1,62 @@
 # Description: Deepspeech native client library.
 
-cc_library(
-    name = "deepspeech",
-    srcs = ["deepspeech.cc"],
-    hdrs = ["deepspeech.h"],
-    deps = ["//tensorflow/core:core",
-            ":c_speech_features"]
-)
+cc_binary(
+    name = "libdeepspeech.so",
+    srcs = ["deepspeech.cc", "deepspeech.h"],
+    deps = [":c_speech_features",
+            "//tensorflow/core:core_cpu",
```

Comment:
Plus, "core_cpu" limits ourselves to no GPU support. Is that what we really want?